plugin/source/main.cpp

```diff
@@ -91,6 +92,10 @@ INITIALIZE_PLUGIN()
             WUPS_StoreInt(nullptr, "quickSelectCombo", currentQuickSelectCombination);
         }
 
+        if ((err = WUPS_GetInt(nullptr, "toggleEmulationCombo", (int32_t*) &currentToggleEmulationCombination)) == WUPS_STORAGE_ERROR_NOT_FOUND) {
+            WUPS_StoreInt(nullptr, "quickSelectCombo", currentToggleEmulationCombination);
```

Comment:
Small copy paste issue:
```suggestion
            WUPS_StoreInt(nullptr, "toggleEmulationCombo", currentToggleEmulationCombination);
```